.github/workflows/deploy_backend_aws_dev.yml

```diff
@@ -84,6 +84,7 @@ jobs:
             EMAIL_VERIFICATION_SECRET=${{ secrets.EMAIL_VERIFICATION_SECRET }}
             NM_MAIL=${{ secrets.NM_MAIL }}
             NM_PASS=${{ secrets.NM_PASS }}
+            CLOUD_URL=${{ vars.FRONTEND_URL }}
 
```

Comment:
Seems like you'll need `DATABASE_URL` here now.

Reply:
Ahh, thanks for catching all these! Have updated all the changes locally. Will update the AWS stack template soon! 👍